tests/Unit/ActionTesting.hpp

```diff
@@ -953,7 +953,7 @@ class MockRuntimeSystem {
     return tuples::get<MockDistributedObjectsTag<Component>>(local_algorithms_);
   }
 
-  const GlobalCache& cache() noexcept { return cache_; }
+  GlobalCache& cache() noexcept { return cache_; }
```

Comment:
Why do you need this?

Reply:
Something about Charm proxies or something.  I didn't change that part of the code.  I'll check if it works with the cache const.

Reply:
I just talked with @wthrowe in the office about this. Here's what Charm++ does.

1. Entry methods should be `const` or non-`const` just like local member functions.
2. As a result, we need non-const proxies (which you can only get from a non-const cache) to invoke entry methods (sometimes).
3. This is made more complicated by Charm++ having a bug (I have not taken the time to track down where in their code) that makes groups and nodegroups be invokable always even with const proxies (or maybe it's arrays, but groups and nodegroups work correctly, I can't remember). My point is we need non-const access to the global cache.